meson_options.txt

```diff
@@ -19,8 +31,7 @@ option(
 )
 
 option(
-  'install_completions',
-  description: 'Install bash, zsh and fish command completions',
-  type: 'boolean',
-  value: true,
+  'zsh_completions_dir',
+  description: 'Directory for Z shell completion scripts (default=$datadir/zsh/site-functions)',
```

Comment:
Is it not possible to use `get_option('data_dir')` here so that the `value` option in `option()` can be used?

Reply:
You mean, to use `get_option('data_dir')` in `meson_options.txt`, so that that the right value shows up in `meson configure`?  That I don't know at all.  I will try.

Or did you mean something else?

Reply:
Yes, I meant that.

Reply:
This doesn't seem to work.

```
meson_options.txt:35:78: ERROR: Expecting rparen got id.
  description: 'Directory for Z shell completion scripts (default=get_option('datadir')/zsh/site-functions)',
                                                                              ^
For a block that started at 33,6
option(
      ^
```

```
meson_options.txt:35:77: ERROR: Expecting rparen got id.
  description: 'Directory for Z shell completion scripts default=get_option('datadir')/zsh/site-functions',
                                                                             ^
For a block that started at 33,6
option(
      ^
```

Reply:
I wish I could somehow use `value` to represent the default value, so that we don't have the maintain default value in two places - one in `meson_options.txt` and then in `meson.build`.

However, it's a lot easier to compare against `''`, than check for a `$` prefix.  Even if it's a `$` prefix, we won't know if it's `$datadir` or some other variable.  It could also not be a variable but the beginning of a legitimate path, because this works:
```
$ touch $
```

So, I am inclined to give up.